bodyprocessors/xml.go

```diff
@@ -4,58 +4,51 @@
 package bodyprocessors
 
 import (
-	"encoding/xml"
 	"io"
-	"strings"
 
+	"github.com/antchfx/xmlquery"
+	"github.com/corazawaf/coraza/v3/internal/collections"
 	"github.com/corazawaf/coraza/v3/rules"
 )
 
+var xmlOptions = xmlquery.DecoderOptions{
+	Strict: false,
+	// AutoClose: xml.HTMLAutoClose,
+	// Entity:    xml.HTMLEntity,
+}
+
```

Comment:
Can we make this configurable? Otherwise, people that want to use it in strict mode can't without recompiling.

Reply:
We already had this discussion in Valencia; let's wait for a feature request on this. It would otherwise require a directive and it would require a huge global refactor